tests/MageTest/Command/FactoryTest.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace MageTest\Command;
+
+use Mage\Command\Factory;
+use PHPUnit_Framework_TestCase;
+
+/**
+ * @group Mage_Command
+ * @group Mage_Command_Factory
+ *
+ * @group issue-167
+ */
+class FactoryTest extends PHPUnit_Framework_TestCase
+{
+    private $config;
+
+    protected function setUp()
+    {
+        $this->config = $this->getMock('Mage\Config');
+    }
+
+    public function testGet()
+    {
+        $command = Factory::get('add', $this->config);
+        $this->assertInstanceOf('Mage\\Command\\BuiltIn\\AddCommand', $command);
+    }
+
+    /**
+     * @expectedException \Exception
+     */
+    public function testGetClassNotFoundException()
+    {
+        $command = Factory::get('commanddoesntexist', $this->config);
+    }
+
+    public function testGetCustomCommand()
+    {
+        $this->getMockBuilder('Mage\\Command\\AbstractCommand')
+            ->setMockClassName('MyCommand')
+            ->getMock();
+
+        /**
+         * current workaround
+         * @see https://github.com/sebastianbergmann/phpunit-mock-objects/issues/134
```

Comment:
use `@link` instead of `@see`.
`@see` is for documentation links, like other methods, classes, etc.
`@link` is purely for hyperlinks